<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellMain</class>
 <widget class="QMainWindow" name="CellMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[name=&quot;btn&quot;](border-radius:5px;background:#409eff;color:white;font-size:15px;)
QLineEdit[name=&quot;le&quot;](border-radius:4px;min-height:25px;border:1px solid gray;)
QLable[name=&quot;lb&quot;](font: 18px &quot;楷体&quot;;)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <widget class="QWidget" name="tools" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>571</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(84,92,100)</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="btn_history">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none; background-color:rgb(84,92,100);color:white;min-height:55px;font:14px &quot;微软雅黑&quot;;padding-left:12px;</string>
       </property>
       <property name="text">
        <string>  借阅记录</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/tool.svg</normaloff>:/tool.svg</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="btn_user_mgr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none; background-color:rgb(84,92,100);color:white;min-height:55px;font:14px &quot;微软雅黑&quot;;padding-left:12px;</string>
       </property>
       <property name="text">
        <string>  用户管理</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/tool.svg</normaloff>:/tool.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btn_book_mgr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none; background-color:rgb(84,92,100);color:white;min-height:55px;font:14px &quot;微软雅黑&quot;;padding-left:12px;</string>
       </property>
       <property name="text">
        <string>  图书管理</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/tool.svg</normaloff>:/tool.svg</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>841</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
